src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  &__img {
+    align-self: center;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    font-size: $font-size--title;
+    font-weight: $font-weight--title;
+    line-height: $line-height--accent;
+    color: $text-color--main;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    line-height: 14px;
+    color: $text-color--secondary;
+  }
+
+  &__rating {
+    margin-top: 16px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    line-height: 14px;
+
+    color: $text-color--main;
+  }
+
+  &__price {
+    margin-top: 24px;
+
+    display: flex;
+    justify-content: space-between;
+
+    line-height: $line-height--accent;
+  }
+
+  &__button-buy {
+    margin-top: 16px;
+    padding: 11px;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    font-size: 14px;
+    font-weight: $font-weight--accent;
+    line-height: $line-height--button;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    color: #fff;
+    background-color: #00acdc;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+
+    transition: 0.5s;
+
+    &:hover {
+      color: #00acdc;
+      background-color: #fff;
+    }
+  }
+}
+
+.price-text {
+  margin: 0;
+
+  font-size: $font-size--title;
+  line-height: $line-height--accent;
+  color: $text-color--secondary;
+}
+
+.price-amount {
+  margin: 0;
+
+  font-size: $font-size--accent;
+  font-weight: $font-weight--accent;
+  line-height: $line-height--accent;
+  color: $text-color--main;
+}
```

Comment:
1 - it should be BEM elements instead of blocks
2 - you can't use margins for BEM blocks

Reply:
done